tests/config1.toml

```diff
@@ -46,8 +46,10 @@ use = ""
 [redis]
 database = 0
 endpoint = "localhost:6379"
+username = ""
 password = ""
 timeout_seconds = 30
+use_clowder = true
```

Comment:
Why we need this? Whether clowder is enabled or not should be done at runtime, not from the config

Reply:
I think this is because the current cache-writer uses clowder but not the inmemorydb from clowder. In any case, as the cache-writer defines no inmemorydb, we don't need the `use_clowder` but rather a "is inmemorydb in clowder config?"

Reply:
True, I'll remove that flag :) The intention was as Juan described.